application/console/migrations/m230515_162100_create_session_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use yii\db\Migration;
+
+class m230515_162100_create_session_table extends Migration
+{
+    // Use safeUp/safeDown to run migration code within a transaction
+    public function safeUp()
+    {
+        /*
+         * UserTable
+         */
+        $this->createTable(
+            '{{session}}',
+            [
+                'id' => 'char(40) primary key not null',
```

Comment:
```suggestion
                'id' => 'char(64) primary key not null',
```
In case we decide to use a hashing algorithm other than MD5.